src/game.zig

```diff
@@ -385,6 +388,27 @@ pub const Player = struct { // MARK: Player
 		return eyePos + super.pos + desiredEyePos;
 	}
 
+	pub fn applyViewBobbingPosBlocking(pos: Vec3d) Vec3d {
+		mutex.lock();
+		defer mutex.unlock();
+		// Horizontal Component - atan(0.25 * sin(bobTime)) / atan(0.25)
+		const xBob = 1.0204970377 * @sin(bobTime);
+		// Vertical Component (*0.8) - atan(0.5 * -sin(2 * bobTime)) / atan(0.5) + 0.5 * (-@sin(bobTime))^2
+		const a = 0.5 * -@sin(2 * bobTime);
+		const b = -@sin(bobTime);
+		const zBob = ((a - a * a * a / 3) / 0.4636476090 + 0.5 * b * b) * 0.8;
```

Comment:
I've updated it to just multiply by 2 instead of dividing by a constant. Not much visual difference and if anything looks a bit better to me, and keeps it simple.